pkg/models/mto_category.go

```diff
@@ -29,6 +31,7 @@ func NewMTOCategory(createdBy uuid.UUID, name string, modelPlanID uuid.UUID, par
 
 // MTOUncategorized returns a placeholder category to hold all milestones that aren't categorized into a subcategory
 func MTOUncategorized(modelPlanID uuid.UUID, parentID *uuid.UUID) *MTOCategory {
+	//TODO (mto) should we add a position? Or ok leaving this as the base implementation?
```

Comment:
What's the effect here? Is `order` returned as `0` for Uncategorized?

Reply:
Yes, it's a 0. It's returned in the correct sequence, but the position doesn't match. Thinking about this more, I think we probably should handle this. I'll make a quick fix

Reply:
Added this in [61f810b](https://github.com/CMS-Enterprise/mint-app/pull/1511/commits/61f810b7ab6f104543f3a514be31449bc052e36b)